warp10/src/main/sh/package.sh

```diff
@@ -28,60 +30,66 @@ WARP10_HOME=warp10-${VERSION}
 ARCHIVE=${WARP_ROOT_PATH}/archive
 
 # Remove existing archive dir
-rm -rf ${ARCHIVE}
-
-# Create the directory hierarchy
-mkdir ${ARCHIVE}
-cd ${ARCHIVE}
-mkdir -p ${WARP10_HOME}/bin
-mkdir -p ${WARP10_HOME}/calls
-mkdir -p ${WARP10_HOME}/datalog
-mkdir -p ${WARP10_HOME}/datalog_done
-mkdir -p ${WARP10_HOME}/etc/bootstrap
-mkdir -p ${WARP10_HOME}/etc/conf.d
-mkdir -p ${WARP10_HOME}/etc/throttle
-mkdir -p ${WARP10_HOME}/etc/trl
-mkdir -p ${WARP10_HOME}/jars
-mkdir -p ${WARP10_HOME}/leveldb/snapshots
-mkdir -p ${WARP10_HOME}/lib
-mkdir -p ${WARP10_HOME}/logs
-mkdir -p ${WARP10_HOME}/macros
-mkdir -p ${WARP10_HOME}/templates
-mkdir -p ${WARP10_HOME}/runners/test/60000
+rm -rf "${ARCHIVE}"
 
+##
+## Create the directory hierarchy
+##
+mkdir "${ARCHIVE}"
+cd "${ARCHIVE}"
+mkdir -p "${WARP10_HOME}/bin"
+mkdir -p "${WARP10_HOME}/calls"
+mkdir -p "${WARP10_HOME}/datalog-ng"
+mkdir -p "${WARP10_HOME}/etc/bootstrap"
+mkdir -p "${WARP10_HOME}/etc/conf.d"
+mkdir -p "${WARP10_HOME}/etc/throttle"
+mkdir -p "${WARP10_HOME}/etc/trl"
+mkdir -p "${WARP10_HOME}/jars"
+mkdir -p "${WARP10_HOME}/lib"
+mkdir -p "${WARP10_HOME}/logs"
+mkdir -p "${WARP10_HOME}/macros"
+mkdir -p "${WARP10_HOME}/tokens"
+mkdir -p "${WARP10_HOME}/runners/test/60000"
+mkdir -p "${WARP10_HOME}/runners/sensision/60000"
 
-cd ${ARCHIVE}
-# Copy startup scripts
-sed -e "s/@VERSION@/${VERSION}/g" ../src/main/sh/warp10.service >> ${WARP10_HOME}/bin/warp10.service
-sed -e "s/@VERSION@/${VERSION}/g" ../src/main/sh/warp10.init >> ${WARP10_HOME}/bin/warp10.init
-sed -e "s/@VERSION@/${VERSION}/g" ../src/main/sh/snapshot.sh >> ${WARP10_HOME}/bin/snapshot.sh
-sed -e "s/@VERSION@/${VERSION}/g" ../src/main/sh/warp10.sh >> ${WARP10_HOME}/bin/warp10.sh
-sed -e "s/@VERSION@/${VERSION}/g" ../src/main/sh/warp10-env.sh >> ${WARP10_HOME}/bin/warp10-env.sh
+##
+## Copy startup scripts
+##
+sed -e "s/@VERSION@/${VERSION}/g" ../src/main/sh/warp10-env.sh >"${WARP10_HOME}/bin/warp10-env.sh"
+sed -e "s/@VERSION@/${VERSION}/g" ../src/main/sh/warp10.service >"${WARP10_HOME}/bin/warp10.service"
+sed -e "s/@VERSION@/${VERSION}/g" ../src/main/sh/warp10.sh >"${WARP10_HOME}/bin/warp10.sh"
```

Comment:
Why is VERSION required in warp10.sh? It reads warp10-env.sh so it should use the value found there.

Reply:
VERSION is now only defined in warp10-env.sh